tests/Feature/PhotosTest.php

```diff
@@ -277,4 +323,11 @@ public function testImport()
 
 		AccessControl::logout();
 	}
+
+	private function asArray($response)
+	{
+		$content = $response->getContent();
+
+		return json_decode($content);
```

Comment:
The method should not be named `asArray`, because `json_decode` actually returns an object not an array. However, IMHO the method is not necessary at all. See comment above.